inferelator_ng/design_response_R.py

```diff
@@ -3,14 +3,11 @@
 """
 
 import os
-import subprocess
```

Comment:
I'm confused about that too.

I think it's harmless unless github reports a conflict.

On Wed, Jul 6, 2016 at 9:49 AM, Nick de Veaux <notifications@github.com>
wrote:

> In inferelator_ng/design_response_R.py
> <https://github.com/simonsfoundation/inferelator_ng/pull/13#discussion_r69732950>
> :
>
> > @@ -3,14 +3,11 @@
> >  """
> >
> >  import os
> > -import subprocess
>
> I'm confused why this is showing up in this pull request diff, since these
> changes have already been made on the master branch: bbf2ca4
> <https://github.com/simonsfoundation/inferelator_ng/commit/bbf2ca4b316d567ad2bddb1088028e67e31d18fe>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/simonsfoundation/inferelator_ng/pull/13/files/0e4b13090ac91f01887c508d11da45779ae42cf2#r69732950>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANNx5ETN3YcmrFUNZtFe4EpRPUssPFNwks5qS7JtgaJpZM4JBZTn>
> .
>